### DISCUSSION TITLE
How to configure telescope to add file limit/timeouts?

### DISCUSSION THREAD

**Discussion by DominickVale** [LazyVim/LazyVim#4786] [General]
Hi. I'm trying to add a file limit to telescope, since lazyvim crashes every time some minified js file ends up being previewed by telescope (or even just opened... but I guess that's another issue. The most important right now is telescope, since it makes searching anything impossible).
This is what I tried, which isn't working.
```lua
  {
    "nvim-telescope/telescope.nvim",
    opts = {
      defaults = {
        file_ignore_patterns = {
          "%.git/.*",
          "node_modules/.*",
        },
        file_previewer = {
          filesize_limit = 0.1
        },
        timeout = 5000,
        preview = {
          timeout = 2000,
        },
      },
    },
  },
```

I tried checking the help pages, but apparently there's no :h telescope.defaults.preview. There seems to be only telescope-native-fzf-[stuff I don't know]

**Comment by dpetka2001** [comment]
The help docs are not available until the plugin is loaded. So, just press `<leader>space` to load Telescope and then `:h telescope.defaults.preview` will be available. Also from the help pages it seems that `filesize_limit` should be under `preview` and not `file_previewer` like you have in your configuration.

**Reply by DominickVale** [reply]
> The help docs are not available until the plugin is loaded. So, just press `<leader>space` to load Telescope and then `:h telescope.defaults.preview` will be available. Also from the help pages it seems that `filesize_limit` should be under `preview` and not `file_previewer` like you have in your configuration.

Thank you very much
